src/components/ListView.js

```diff
@@ -113,6 +113,28 @@ const ListView = ({ shoppingList, loading, error }) => {
               );
             })}
           </ul>
+          <div className="legend" aria-hidden="true">
```

Comment:
There's something weird with the legend, it's squashed?
[
![legend](https://user-images.githubusercontent.com/16920088/118310977-fa0cd980-b4f7-11eb-89ea-7ef968828ffb.png)
](url)

Reply:
The problem is not with the legend, the CSS of the nav-links just needs to be adjusted. I believe such styling concerns will be handled with next week's issue. Thank you.

Reply:
The problem is not with the legend, the CSS of the nav-links just needs to be adjusted. I believe such styling concerns will be handled with next week's issue. Thank you.

Reply:
I've set the navbar to ``display:none`` , the legend box is still squashed
![legend2](https://user-images.githubusercontent.com/16920088/118331323-22550200-b511-11eb-9dba-b20bc06cf1e8.png)

Reply:
I thought I edited the message, sorry. The problem is not the navbar. That particular box is squashed because it's not responsive yet (there's a lot of text), and that is, like I said earlier, a CSS problem, not a functionality problem. Design is not a priority at the moment, and this can be fixed next week.

Do you mind approving the PR if there are no other issues?

Reply:
Adding the legend was a nice touch but it wasn't necessary for this week :slightly_smiling_face:. So no need to focus on it's CSS for now. It can even be deleted if it's causing trouble. 
Did you test if everything else was working properly before adding it? Adding such extra things can cause unnecessary strain and throw off project estimations.

Reply:
Thank you, Golfo.
I agree that it wasn't necessary, but it was requested during reviews and was holding the process up, so I just went ahead and added it.

Yes, everything else works properly.